redis/src/lib.rs

```diff
@@ -143,6 +153,23 @@ impl Manager {
         Ok(Self {
             client: Client::open(params)?,
             ping_number: AtomicUsize::new(0),
+            connection_config: AsyncConnectionConfig::default(),
+        })
+    }
+
+    /// Creates a new [`Manager`] from the given `params` and [`AsyncConnectionConfig`].
+    ///
+    /// # Errors
+    ///
+    /// If establishing a new [`Client`] fails.
+    pub fn new_with_connection_config<T: IntoConnectionInfo>(
```

Comment:
I think I would like the name `from_config` or `from_async_connection_config` better for this. What do you think?

Reply:
`from_config` seemed more consistent with how things are named elsewhere, so I've changed it to that 👍